web/views.py

```
@@ -6955,3 +6973,448 @@ def delete_post(request, post_id):
     if request.method == "POST":
         post.delete()
     return redirect("social_media_dashboard")
+
+
+@login_required
+def manage_membership(request) -> HttpResponse:
```

Ruff (0.8.2)

Missing type annotation for function argument `request`

(ANN001)